include/bluetooth/scan.h

```diff
@@ -290,6 +290,13 @@ struct bt_scan_device_info {
 
 	/** Connection parameters for LE connection. */
 	const struct bt_le_conn_param *conn_param;
+
+	/** Received advertising data. If futher
+	 *  data proccesing is needed, you should
+	 *  use @ref bt_data_parse to get specific
+	 *  advertising data type.
+	 */
+	struct net_buf_simple *adv_data;
```

Comment:
why is this field necessary? you don't use it in the central_hids sample

Reply:
In context of this PR it is not required or used. I just think that is worth to have this as new feature if sameone would like to parse other data in advertising packets, without it scanning module appropriates adv data.

Reply:
@KAGA164 Have you had a look at the new scan callbacks that was added?

Have a look here: https://github.com/zephyrproject-rtos/zephyr/blob/master/include/bluetooth/bluetooth.h#L547

Reply:
I saw this multiplexing scaning callback. Here I added it for case when you want to use scanning module and you also want to parse other data which scanning module does not compare. Scanning module handle filtering by (address, name, short name, appearance, uuid and manufacturer data) but adv packets can contains another type of data. That is why I think that maybe is worth to pass this data to application

Reply:
Very well. Just wanted to make sure. It makes sense for the completeness of this module I agree.